### PR TITLE
MAG2x: Changing long array syntax to the short syntax one.

### PR DESCRIPTION
**Pull request by guzzilar**
> ⚠️ **Note, this pull request requires PR #123 to be merged first.**

#### 1. Objective

screen shot 2561-09-18 at 14 54 14

There are code standard provided by Magento that all extensions have to follow in order to submit the code to their Magento Marketplace.

This pull request is aiming to update one of the standard (array syntax) to a proper one that Magento requires.

#### 2. Description of change

- Update all long-syntax-array to the short one.

#### 3. Quality assurance

1. Install https://github.com/magento/marketplace-eqp.
2. Execute the command  `vendor/bin/phpcs /path/to/your/extension --standard=MEQP2`.
3. Search for a warning message `Short array syntax must be used to define arrays`.

**Before change, we should see these ones**
screen shot 2561-09-18 at 14 54 14

**After change, we should see none of them**
screen shot 2561-09-18 at 15 01 40

#### 4. Impact of the change

Nothing in a user perspective, but it will make our extension be able to pass the Magento Code Quality Standard and able to submit to the Magento Marketplace.

#### 5. Priority of change

Normal

#### 6. Additional Notes

-

### CODE REVIEW

File: Test/Unit/Model/Api/ObjectTest.php [hunk 1]
```diff
@@ -98,7 +98,7 @@ public function reload()
 class MockOmiseObject implements \ArrayAccess
 {
     // Store the attributes of the object.
-    protected $_values = array();
+    protected $_values = [];
```
Comment (jonrandy) [inline Test/Unit/Model/Api/ObjectTest.php]: Protected var name starting with `_`

Comment (guzzilar) [reply Test/Unit/Model/Api/ObjectTest.php]: @jonrandy Thanks
Code-refactoring PR will be coming in soon!

Comment (jacekstan) [reply Test/Unit/Model/Api/ObjectTest.php]: @guzzilar no necessary to change in pr only few places.. change all in entire plugin..

Comment (guzzilar) [reply Test/Unit/Model/Api/ObjectTest.php]: @jacekstan yup, so changing entire plugin will be coming in an another pull request once `tesco-lotus` feature is merged (or maybe before) (I updated array syntax & opened this pull request just to continue from a conversation at the previous PR #123)

Comment (jacekstan) [reply Test/Unit/Model/Api/ObjectTest.php]: so why this PR *now* is necessary?
and why there is problem to follow all plugin with those changes?

**Pull request merged**